pandas/io/sas/sas_xport.py

```diff
@@ -139,7 +140,7 @@
 """
 
 
-def _parse_date(datestr: str) -> datetime:
+def _parse_date(datestr: str) -> datetime | NaTType:
```

Comment:
I thought we have a type that is a datetime scalar | NatType already?

Reply:
probably not bc ATM NaTType subclasses datetime